src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -386,15 +392,37 @@ void setOptimalIndex(int optimalIndex)
 
         RawPacket[] pktsOut = bitstreamController.rtpTransform(pktIn);
 
-        if (!ArrayUtils.isNullOrEmpty(pktsOut)
-            && pktIn.getSSRCAsLong() != targetSSRC)
+        if (REMOVE_VP8_EXTENDED_CONTROL_BITS
+            || (!ArrayUtils.isNullOrEmpty(pktsOut)
```

Comment:
What is the idea here? Why do we want to execute the block if e.g. pktsOut == null && REMOVE_VP8_EXTENDED_CONTROL_BITS?